components/eam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -1048,6 +1091,29 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
        nccn_prescribed = ccn_trcdat*cld_frac_l
     end if
 
+    ! Write to outfile the variables needed for p3-stand-alone initialization
+    call outfld("T_mid_inP3",              state%T,                pcols, lchnk) 
+    call outfld("p_mid_inP3",              state%pmid,             pcols, lchnk) 
+    call outfld("pseudo_density_inP3",     state%pdel  ,           pcols, lchnk) 
+    call outfld("qc_inP3",                 state%q(:,:,ixcldliq),  pcols, lchnk) 
+    call outfld("qv_inP3",                 state%q(:,:,1),         pcols, lchnk) 
+    call outfld("z_int_inP3",              state%zi,               pcols, lchnk) 
+    call outfld("inv_qc_relvar_inP3",      relvar,                 pcols, lchnk) 
+    call outfld("bm_inP3",                 state%q(:,:,ixrimvol),  pcols, lchnk) 
+    call outfld("nc_inP3",                 state%q(:,:,ixnumliq),  pcols, lchnk) 
+    call outfld("ni_inP3",                 state%q(:,:,ixnumice),  pcols, lchnk) 
+    call outfld("nr_inP3",                 state%q(:,:,ixnumrain), pcols, lchnk) 
+    call outfld("qi_inP3",                 state%q(:,:,ixcldice),  pcols, lchnk) 
+    call outfld("qm_inP3",                 state%q(:,:,ixcldrim),  pcols, lchnk) 
+    call outfld("qr_inP3",                 state%q(:,:,ixrain),    pcols, lchnk) 
+    call outfld("cldfrac_tot_inP3",        ast,                    pcols, lchnk) 
+    call outfld("cldfrac_tot_inRAD",       ast,                    pcols, lchnk) ! Radiation needs this one too in SCREAM
```

Comment:
If radiation is using this, shouldn't it be output from radiation?

Reply:
I don't think radiation is using this in F90.  I'll have to check.  If it isn't then I figured it was probably more than we needed to have radiation grab it from the pbuf just to output it there.  That being said it is a little weird to grab something for RAD from P3.  So maybe I should just add it in the RAD interface anyways.